### PR TITLE
Feature/deployment logs

### PR DESCRIPTION
**Pull request by zelima**
- See deployment logs from Drone CI/CD 
- Check helm release status
- Check deployment version

WIP: Manage images

### CODE REVIEW

File: ckan_cloud_operator/providers/ckan/deployment/drone/manager.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+    def initialize(self, force_prompt=False):
+        self.conf = env_manager._read_yaml().get('cicd', {})
+        self.server_url = self.conf.get('serverUrl')
+        self.api_token = self.conf.get('token')
+        self.org = self.conf.get('organization')
+        self.repo = self.conf.get('repo')
+        if force_prompt or not self.conf:
+            self.server_url = input('Please enter Drone Server URL: ')
+            self.api_token = input(f'Please enter Drone API Tocken. See {self. server_url}acount: ')
```
Comment (pdelboca) [inline ckan_cloud_operator/providers/ckan/deployment/drone/manager.py]:
```suggestion
            self.api_token = input(f'Please enter Drone API Token. See {self. server_url} acount: ')
```

File: ckan_cloud_operator/providers/ckan/deployment/drone/manager.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            env_manager._write_yaml(self.conf)
+        self.api_url = urljoin(self.server_url, '/'.join(['api/repos', f'{self.org}/{self.repo}', 'builds/']))
+        self.header = {'Authorization': f'Bearer {self.api_token}'}
+        print('CCO is configured for Drone ')
+
+
+    def builds_list(self):
+        resp = requests.get(self.api_url, headers=self.header)
+        if not _check_for_200(resp):
+            return []
```
Comment (pdelboca) [inline ckan_cloud_operator/providers/ckan/deployment/drone/manager.py:35]:
```suggestion
            return {}
```
Better return an empty json if the expected return value is a json object.

File: ckan_cloud_operator/providers/ckan/env/manager.py [hunk 3]
```diff
@@ -163,6 +163,13 @@ def _write_yaml(data):
     _file.close()
 
 
+def _read_yaml():
+    _file = open(_get_config_file_name(), "r")
+    yaml_conf = yaml.load(_file, Loader=yaml.FullLoader)
+    _file.close()
+    return yaml_conf
```
Comment (pdelboca) [inline ckan_cloud_operator/providers/ckan/env/manager.py]:
```suggestion
def _read_yaml():
    with open(_get_config_file_name()) as _file:
        yaml_conf = yaml.load(_file, Loader=yaml.FullLoader)
    return yaml_conf
```

It is always suggested as good practice to use `with` statement when reading files, also "r" is the default mode so not need to add it :)

**Review by pdelboca — COMMENTED** [review]
Looks good @zelima ! I left a few comments.

What is the goal of `ckan_cloud_operator/drone.py` file? It seems like a duplicated version of ` ckan_cloud_operator/providers/ckan/deployment/drone/manager.py`

File: ckan_cloud_operator/providers/ckan/deployment/drone/manager.py [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+                return str(build.get('number'))
+        print(f'No build found for branch {branch}')
+        return None
+
+
+def _check_for_200(resp):
+    if resp.status_code == 200:
+        return True
+    if resp.status_code == 401:
+        print("Seems like you are not authorised")
```
Comment (pdelboca) [inline ckan_cloud_operator/providers/ckan/deployment/drone/manager.py]:
```suggestion
        print("Seems like you are not authorized")
```

File: ckan_cloud_operator/providers/ckan/deployment/drone/manager.py [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+        print("Please run `cco ckan deployment drone initialize --force-update` and rerun")
+        return False
+    print(resp.text)
```
Comment (pdelboca) [inline ckan_cloud_operator/providers/ckan/deployment/drone/manager.py]: There is no need to print the resp text each time we check the status.
```suggestion
```

File: ckan_cloud_operator/providers/ckan/deployment/drone/manager.py (hunk 2 above)
Comment (zelima) [reply ckan_cloud_operator/providers/ckan/deployment/drone/manager.py:35]: response is a list of build info. So this needs to be list

**Pull request merged**